msal_extensions/persistence.py

```diff
@@ -121,7 +121,10 @@ def save(self, content):
     def load(self):
         # type: () -> str
         with open(self._location, 'rb') as handle:
-            return self._dp_agent.unprotect(handle.read())
+            data = handle.read()
+            if data is not b"":
```

Comment:
```suggestion
            if data:
```

Alternatively, if you are in a geeky mood:

> return self._dp_agent.unprotect(data) if data else None